core/keyboard_nav/cursor_svg.js

```diff
@@ -511,6 +511,12 @@ Blockly.CursorSvg.prototype.draw = function(curNode) {
   } else if (curNode.getType() === Blockly.ASTNode.types.STACK) {
     this.showWithStack_(curNode);
   }
+
+  // Ensures the cursor will be visible immediately after the move.
+  var animate = this.currentCursorSvg.children[0];
```

Comment:
Node.children is unavailable on IE 11. Use childNodes instead.

Reply:
ah thanks good to know!